src/app/components/ReadTime/index.tsx

```diff
@@ -5,19 +5,45 @@ import isLive from '#app/lib/utilities/isLive';
 import { ServiceContext } from '#app/contexts/ServiceContext';
 import { EventTrackingData } from '#app/lib/analyticsUtils/types';
 import useViewTracker from '#app/hooks/useViewTracker';
+import Text from '#app/components/Text';
 import styles from './index.styles';
 
 type ReadTimeProps = {
-  readTime: number;
+  readTime?: number;
   className?: string;
   readTimeVariant?: string;
 };
 
+const DEFAULT_TRANSLATIONS = {
+  readTimePrefix: 'Estimated Read Time',
+  quick: 'Quick Read',
+  long: 'Long Read',
+};
+
 const ReadTime = ({ readTime, readTimeVariant, className }: ReadTimeProps) => {
-  if (isLive() || readTimeVariant === 'off' || !readTimeVariant) return null;
+  if (!readTime || isLive() || readTimeVariant === 'off' || !readTimeVariant)
+    return null;
+
+  const { translations } = use(ServiceContext);
+  const readTimePrefix =
+    translations.readTime?.readTimePrefix ??
+    DEFAULT_TRANSLATIONS.readTimePrefix;
+  const quickCopy = translations.readTime?.quick ?? DEFAULT_TRANSLATIONS.quick;
+  const longCopy = translations.readTime?.long ?? DEFAULT_TRANSLATIONS.long;
+
+  // EXPERIMENT: Read Time
+  const fontSize = readTimeVariant.includes('bold') ? 'pica' : 'brevier';
+  const fontVariant = readTimeVariant.includes('bold')
+    ? 'sansBold'
+    : 'sansRegular';
+  const readTimeCopyType = readTimeVariant.includes('minutes')
+    ? 'minutes'
+    : 'quickLong';
 
   const readTimeInMiliseconds = readTime * 60000;
   const minutesLabel = readTime === 1 ? 'minute' : 'minutes';
+  const quickLongCopy = readTime < 5 ? quickCopy : longCopy;
+  const minutesCopy = `${readTimePrefix}: ${readTime} ${minutesLabel}`;
```

Comment:
Minor naming suggestion, feel free to ignore.
```suggestion
  const shortCopy = readTime < 5 ? quickCopy : longCopy;
  const verboseCopy = `${readTimePrefix}: ${readTime} ${minutesLabel}`;
```

Reply:
We went with naming used in the Experiment Brief - https://paper.dropbox.com/doc/Experiment-Brief-WS-Article-Read-Time--Cr_k1aB2aIjZpObbtJsm86lwAg-kaU1pL80CpbNcf7nOyM06#:uid=430117666644784764994442&h2=Experiment-solution: